src/FSharp.Core/list.fsi

```diff
@@ -1686,6 +1883,9 @@ module List =
     val pick: chooser:('T -> 'U option) -> list:'T list -> 'U
 
     /// <summary>Returns a list with all elements permuted according to the
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(n)
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence